common/channelconfig/util.go

```diff
@@ -239,53 +239,58 @@ func ACLValues(acls map[string]string) *StandardConfigValue {
 	}
 }
 
-// ValidateCapabilities validates whether the peer can meet the capabilities requirement in the given config block
-func ValidateCapabilities(block *cb.Block, bccsp bccsp.BCCSP) error {
+// ExtractChannelConfig extracts channel config from a block
+func ExtractChannelConfig(block *cb.Block, bccsp bccsp.BCCSP) (*ChannelConfig, error) {
```

Comment:
Nit. For better readability, it is always good to place the caller first and then the callee. In this case, `ValidateCapabilities()` can be placed just before the `ExtractChannelConfig()`.